app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/dashboard.tsx

```diff
@@ -212,95 +202,114 @@ export default function Dashboard(props: DashboardProps) {
         }
     }, [/* should never change */ unsetModal])
 
+    const driveHiddenClass = page === pageSwitcher.Page.drive ? '' : 'hidden'
```

Comment:
there's nowhere else in the codebase that does this, so not sure this should be refactored